operating_procedures.md

```diff
@@ -62,33 +60,47 @@ In collaboration with Editors and Maintainers, Contributors help drive the evolu
 
+Editors are designated by the WG Chair and ratified by the Working Group Members.
+
 ### 2.2.3 Task Force Convener
 
 Task Force Conveners are dedicated Maintainers within the FinOps FOCUS project who take on the responsibility of leading specific technical workstreams or topics within Task Forces. This leadership role involves guiding the development and implementation of specialized areas of the project, ensuring that all activities align with the project's broader objectives. Conveners are essential in driving detailed technical work, fostering collaboration, and maintaining high standards within their respective domains.
 
 
 ### 2.2.4 FOCUS Group Maintainer
   
-  FG Maintainers are a subset of the FOCUS Group members who have been given write access to one or more FOCUS Working Group repositories within the FOCUS Project's Github organization. They will advance the day-to-day evolution of the specification and related work products.
+  FG Maintainers are a subset of the FOCUS Group (FG) members who have been given write access to one or more FOCUS Working Group repositories within the FOCUS Project's Github organization. They will advance the day-to-day evolution of the specification and related work products.
 
   FG Maintainers are responsible for ensuring that the contents of documents and work products accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines. 
 
-  FG Chair(s) will designate a FG Maintainer or FG Maintainers for the FG. FG Maintainer(s) are overseen by of the FG Chair(s).
+  FG Chair(s) designates any FG Maintainer(s) for the FG. FG Maintainers are overseen by the FG Chair(s). Maintainer designations will be revisited at the discretion of the FG chair, typically at the conclusion of a release cycle.  
 
-  The FG Maintainer(s) MUST keep the project documentation up to date (e.g.: contributing, readme and release planning documents).
+  To be designated as a Maintainer, individuals MUST:
 
-  The FG Maintainer(s) MUST apply the "Review & Approval" process—covered in 6.1 of this document—to contributions submitted by the Working Group members.
+   * Be involved as a Contributor consistently over at least one release cycle
```

Comment:
I think this should be 2 or more cycles

Reply:
Disagree.  If someone has a strong showing i/t/o spec contribution over the course of a single release, we should consider designating them; not automatically of course, but they would be eligible.  If two or more cycles were the rule, then I would not have been eligible to be designated a Maintainer after 1.0 (though maybe you would have preferred that!!).